.github/workflows/cd.yml

```diff
@@ -93,11 +93,7 @@ jobs:
           retry_on: error
           shell: pwsh
           command: |
-            $version=Get-Content packages/fx-core/templates/plugins/resource/simpleauth/version.txt
-            $tag = "simpleauth@"+$version
-            $fileName="Microsoft.TeamsFx.SimpleAuth_$version.zip"
-            $url = "https://github.com/OfficeDev/TeamsFx/releases/download/"+$tag+"/"+$fileName
-            Invoke-WebRequest $url -OutFile packages/fx-core/templates/plugins/resource/simpleauth/SimpleAuth.zip
+            ./.github/scripts/downloadSimpleAuth.ps1
```

Comment:
will powershell fail on linux agent?

Reply:
About GitHub-hosted runners - GitHub Docs
virtual-environments/Ubuntu2004-README.md at main · actions/virtual-environments (github.com)
powershell is the default shell installed on images from github